source/locale/zh_CN/LC_MESSAGES/reference/generated/optuna.pruners.MedianPruner.po

```diff
@@ -23,21 +23,21 @@ msgstr ""
 
 #: of optuna.pruners._median.MedianPruner:1
 msgid "Pruner using the median stopping rule."
-msgstr ""
+msgstr "使用中位数终止规则的 pruner."
 
 #: of optuna.pruners._median.MedianPruner:3
 msgid ""
 "Prune if the trial's best intermediate result is worse than median of "
 "intermediate results of previous trials at the same step."
-msgstr ""
+msgstr "如果该 trial 的最佳中间值比之前所有 trial 在同一步骤时的最差中间值还差, 则剪枝."
```

Comment:
如果该 trial 的最佳中间值比之前所有 trial 在同一步骤时的中间值的中位数还差, 则剪枝.

I want to change the word `中间值`。Because the `median` and `intermediate result` are both translated as `中间值`, but of course they are different. It can be confusing

Reply:
Agreed. How about `"如果该 trial 的最佳中间值比之前 trial 在同一步骤时所有中间值的中位数还差, 则剪枝."`?

Reply:
It sounds good to me